app/src/main/res/values/strings.xml

```diff
@@ -312,6 +312,11 @@
     <string name="preferences_screenshots_in_private_mode_disclaimer">If allowed, private tabs will also be visible when multiple apps are open</string>
     <!-- Preference for adding private browsing shortcut -->
     <string name="preferences_add_private_browsing_shortcut">Add private browsing shortcut</string>
+    <string name="errorpage_httpsonly_title">Secure site not available</string>
+    <!-- Message shown in the error page for when trying to access a http website while https only mode is enabled. The message has two paragraphs. This is the first. -->
+    <string name="errorpage_httpsonly_message_title">Most likely, the website simply does not support HTTPs.</string>
+    <!-- Message shown in the error page for when trying to access a http website while https only mode is enabled. The message has two paragraphs. This is the second. -->
+    <string name="errorpage_httpsonly_message_summary">However, it’s also possible that an attacker is involved. If you continue to the website, you should not enter any sensitive info. If you continue, HTTPS-Only mode will be turned off temporarily for the site.</string>
```

Comment:
Can we move these strings to an appropriate section and not inject them in between the `preferences_` strings so that we can maintain keeping related strings to a particular screen together

Reply:
This patch would be added on top of https://github.com/mozilla-mobile/fenix/pull/24095 which adds the strings for these section at lines 321-334 - below the strings for what visually the settings section above - "Private browsing" uses.